app/views/declaration/representative_details.scala.html

```diff
@@ -17,70 +17,92 @@
+                    value = Some(DirectRepresentative.toString),
+                    content = Text(messages("supplementary.representative.representationType.direct")),
+                    hint = Some(Hint(
+                        content = Text(messages("supplementary.representative.representationType.direct.hint"))
+                    )),
+                    checked = form("statusCode").value.contains(DirectRepresentative.toString)
+                ),
+                RadioItem(
+                    id = Some(IndirectRepresentative.toString),
+                    value = Some(IndirectRepresentative.toString),
+                    content = Text(messages("supplementary.representative.representationType.indirect")),
+                    hint = Some(Hint(
+                        content = Text(messages("supplementary.representative.representationType.indirect.hint"))
+                    )),
+                    checked = form("statusCode").value.contains(IndirectRepresentative.toString)
+                )
+            ),
+            errorMessage = form("value").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*))))
+        ))
+
+        @exportsInputText(
+            field = form("details.eori"),
+            labelKey = "supplementary.representative.eori.legend",
+            hintKey = Some("supplementary.representative.eori.hint"),
+            isPageHeading = true,
+            headingClasses = "govuk-label--m"
         )
 
-        <p>@messages("supplementary.representative.hint")</p>
-
-        @components.form_fields_wrapper(field = form("details")) {
-
-            @components.fields.field_text(
-                field = form("details.eori"),
-                label = Some(messages("supplementary.representative.eori.info")),
-                labelClass = Some("heading-24")
-            )
-
-            <p class="mt-5">Or</p>
-
-            @components.address(form, "details.address", Some(messages("supplementary.representative.address.info")))
+        @components.details(messages("site.details.summary_text_these")){
+            @tariffBody
         }
 
-        <h1 class="title" title="title-representation-type">@messages("supplementary.representative.representationType.header")</h1>
-
-        @components.fields.field_radio(
-            field = form("statusCode"),
-            legend = "",
-            inputs = Seq(
-                RadioOption("statusCode_direct", DirectRepresentative,
-                    messages("supplementary.representative.representationType.direct"), Some(messages("supplementary.representative.representationType.direct.hint"))),
-                RadioOption("statusCode_indirect", IndirectRepresentative,
-                    messages("supplementary.representative.representationType.indirect"), Some(messages("supplementary.representative.representationType.indirect.hint")))
-            )
-        )
+        @govukButton(Button(content = Text(messages("site.save_and_continue"))))
```

Comment:
You can replace this and the button below with @saveButtons() - see consignmentReferences view